client/src/components/Ticketing/ticketingmanager/Season/SeasonInstancesPage.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useEffect, useState} from 'react';
+import {useNavigate} from 'react-router-dom';
+import {titleCase} from '../../../../utils/arrays';
+import {useFetchToken} from '../showings/ShowingUpdated/ShowingUtils';
+import {getSeasonImage, formatSeasonDate} from './seasonUtils';
+
+export interface Seasons {
+  seasonid: number;
+  name: string;
+  startdate: number;
+  enddate: number;
+  imageurl?: string;
+}
+
+/**
+ * Display all seasons page
+ *
+ * @module
+ * @returns Season Instances Page
+ */
+const SeasonInstancesPage = () => {
+  const navigate = useNavigate();
+  const [seasons, setAllSeasons] = useState<Seasons[]>([]);
+  const {token} = useFetchToken();
+
+  const getAllSeasons = async () => {
+    try {
+      const getAllSeasonsRes = await fetch(
+        process.env.REACT_APP_API_2_URL + '/season',
+        {
+          credentials: 'include',
+          method: 'GET',
+          headers: {
+            'Content-Type': 'application/json',
+            'Authorization': `Bearer ${token}`,
+          },
+        },
+      );
+
+      if (!getAllSeasonsRes.ok) {
+        throw new Error('Failed to fetch all seasons');
+      }
+
+      const seasonsData = await getAllSeasonsRes.json();
+      setAllSeasons(seasonsData);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    getAllSeasons();
+  }, [token]);
+
+  return (
+    <div className='w-full h-screen overflow-x-hidden absolute'>
+      <div
```

Comment:
I recently made some mobile improvements to the events version of this page, do you think you could copy the classes on this `div` and the elements within from there? Since its a bit smushed at the moment:
![image](https://github.com/WonderTix/WonderTix/assets/92124260/e6cb22f9-7296-4c2d-9b7f-f70206129ed8)

Reply:
Got it! I forgot you had made those changes. Thanks!